File: testprojects/csharp/benchmark/domain/Indirect/AlowedFrom/AccessInstanceVariableIndirect_VarVarToString.cs
namespace Domain.Indirect.AllowedFrom
{
	public class AccessInstanceVariableIndirect_VarVarToString
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

        private string test;

        public void Case3()
        {
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
            test = bckg.serviceOne.day.ToString();
        }
	}
}

File: testprojects/java/accuracy/domain/indirect/allowedfrom/AccessInstanceVariableIndirect_VarVarToString.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class AccessInstanceVariableIndirect_VarVarToString {
	private BackgroundService bckg;
	private String test;

	public AccessInstanceVariableIndirect_VarVarToString(){
		bckg = new BackgroundService();
	}
		// Indirect access of a normal attribute and with toString()
		public String case3() {
			test = bckg.serviceOne.day.toString();
			return test;
		}
}
